app/src/main/AndroidManifest.xml

```diff
@@ -11,8 +11,8 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
-    <activity android:name=".player.exploration.ExplorationActivity"/>
-    <activity android:name="org.oppia.app.home.HomeActivity">
+    <activity android:name=".player.exploration.ExplorationActivity">
+    <!--<activity android:name="org.oppia.app.home.HomeActivity">-->
```

Comment:
Revert these changes, we don't need to make sure that the PR code uses your implementation.
Rather than that we want that your PR should be merged directly and whenever the develop runs the develop branch, homefragment should appear.

Reply:
reverted